packages/docs/operations-manual/key-management/detailed.md

```diff
@@ -0,0 +1,148 @@
+An authorized validator signer key that will be used to register a validator group can be used to send group management transactions (e.g. register, add member A, queue commission update to 0.25, etc.) Because this key does not participate directly in consensus it **can be stored on a Ledger hardware wallet.**
+
+### Authorized Validator Signers for Validators
+An authorized validator signer key that will be used to register a validator can be used to send validator management transactions (e.g. register, affiliate with group A, etc.) This key will also be used to sign consensus messages and thus *cannot be stored on a Ledger hardware wallet* as signing consensus messages is not currently supported by the Celo Ledger App.
+
+Note that the validator signing key must first be used to generate a "proof-of-possession" indicating that key's willingness to be authorized as a signer on behalf of the Locked Gold Account.
+
+
+### Authorizing a Validator Signer
+An undesignated key may be authorized as a validator signer on behalf of a Locked Gold Account by running the following commands:
+
+```shell
+# Create a proof-of-possession. Note that the signer private key must be available.
+# Note that the signing key can be kept on a Ledger if it will be used to run a Validator Group.
+celocli account:proof-of-possession --account $LOCKED_GOLD_ACCOUNT --signer $KEY_TO_AUTHORIZE
+
+# Authorize the validator signing key. Note that the Locked Gold Account private key must be available.
+# Note that if a Validator has previously been registered on behalf of the Locked Gold Account it
+# may be desirable to include the BLS key here as well. Please see the documentation below on validator
+# key rotation for more information.
+celocli account:authorize --from $LOCKED_GOLD_ACCOUNT --role validator --signer $KEY_TO_AUTHORIZE --signature $PROOF_OF_POSSESSION --useLedger
+
+# Confirm that the vote signer was authorized
+celocli account:show $LOCKED_GOLD_ACCOUNT
+
+# You can also look up account info via the authorized signer
+celocli account:show $KEY_TO_AUTHORIZE
+```
+
+## Authorized Validator BLS Signers
+
+The Celo protocol uses BLS signatures in consensus to ultimately determine whether or not a particular block is valid. Many BLS signatures over the same content can be combined into a single "aggregated signature", allowing several kilobytes of signatures to be compressed into fewer than 100 bytes, ensuring that the block headers remain compact and mobile friendly.
+
+When registering a Validator on behalf of a Locked Gold Account, users must provide a BLS public key, as well as a proof-of-possession to protect against [rogue key attacks](https://crypto.stanford.edu/~dabo/pubs/papers/BLSmultisig.html).
+
+By default users can derive the BLS key directly from their authorized validator signer key. From a key management perspective, this means that the authorized BLS signer key is **exactly the same** as the authorized validator signer key.
```

Comment:
security perspective?

Reply:
"key management and security perspective"?

I think folks will want to realize that this isn't exactly another key that they need to custody